### ISSUE TITLE
2.1.0a2 does not install headers

### ISSUE THREAD

**Issue by antonio-rojas** [aleaxit/gmpy#189]
'python setup.py install' does not install *.h or *.pxd headers in 2.1.0a2. Used to work in 2.1.0a1

**Comment by vinklein** [comment]
It seems install-data command has been removed from the setup.py. I don't know why.
You can patch your setup.py to have gmpy2.h and gmpy2.pxd installed. 
```
--- a/setup.py
+++ b/setup.py
@@ -2,6 +2,7 @@ import platform
 import os
 from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext
+from distutils.command.install_data import install_data
 
 ON_WINDOWS = platform.system() == 'Windows'
 _comp_args = ["DSHARED=1"]
@@ -95,7 +96,13 @@ extensions = [
               )
 ]
 
-cmdclass = {'build_ext': Gmpy2Build}
+class gmpy_install_data(install_data):
+    def finalize_options(self):
+        install_data.finalize_options(self)
+        install = self.distribution.get_command_obj('install')
+        self.install_dir = install.install_purelib
+
+cmdclass = {'build_ext': Gmpy2Build, 'install_data' : gmpy_install_data}
```

**Comment by vinklein** [comment]
I am will propose a pull request with the above patch.

**Comment by vinklein** [comment]
The patch don't work right now.

**Comment by vinklein** [comment]
I haven't find a solution right now. After installing `gmpy2.h` and `gmpy2.pxd` under `sites-packages` directory (instead of `sites-packages/gmpy2`) and fixing travis to run cython tests (the current one does not). I got an error :
```
The command "python test/runtests.py" exited with 0.
2.47s$ python test_cython/runtests.py
()
Unit tests for gmpy2 2.1.0a3dev0 with Cython 0.28.2
()
Compiling test_cython.pyx because it changed.
[1/1] Cythonizing test_cython.pyx
running build_ext
building 'test_cython' extension
creating build
creating build/temp.linux-x86_64-2.7
gcc -pthread -fno-strict-aliasing -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages -I/tmp/tmpOi153U -I/home/travis/virtualenv/python2.7.14/lib/python27.zip -I/home/travis/virtualenv/python2.7.14/lib/python2.7 -I/home/travis/virtualenv/python2.7.14/lib/python2.7/plat-linux2 -I/home/travis/virtualenv/python2.7.14/lib/python2.7/lib-tk -I/home/travis/virtualenv/python2.7.14/lib/python2.7/lib-old -I/home/travis/virtualenv/python2.7.14/lib/python2.7/lib-dynload -I/opt/python/2.7.14/lib/python2.7 -I/opt/python/2.7.14/lib/python2.7/plat-linux2 -I/opt/python/2.7.14/lib/python2.7/lib-tk -I/opt/python/2.7.14/include/python2.7 -c test_cython.c -o build/temp.linux-x86_64-2.7/test_cython.o
gcc -pthread -shared -L/opt/python/2.7.14/lib -Wl,-rpath=/opt/python/2.7.14/lib build/temp.linux-x86_64-2.7/test_cython.o -L/opt/python/2.7.14/lib -lgmp -lmpfr -lmpc -lpython2.7 -o /tmp/tmpOi153U/test_cython.so
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "test_cython.pyx", line 5, in init test_cython
    import_gmpy2()
AttributeError: 'module' object has no attribute '_C_API'
cython test failed
```
see full log here https://travis-ci.org/vinklein/gmpy/jobs/367709033

**Comment by vinklein** [comment]
It's fixed with PR #190

**Comment by jdemeyer** [comment]
Why was this closed? This issue isn't fixed at all...

Please re-open.
